packages/cli-utils/package.json

```diff
@@ -49,7 +49,8 @@
   },
   "dependencies": {
     "@gql.tada/internal": "workspace:*",
-    "graphql": "^16.8.1"
+    "graphql": "^16.8.1",
+    "semiver": "^1.1.0"
```

Comment:
Let's move this to `devDependencies` and bundle it away, for the same reasons as we bundle most other things away in `cli-utils`?